### PR TITLE
[feature][doc]Add doc for how to use MultiRolesTokenAuthorizationProvider

### LINKED ISSUE #14741: [feat][doc] add explanations for how to use MultiRolesTokenAuthorizationProvider
## Issue

Currently, we do not have docs on how to use `MultiRolesTokenAuthorizationProvider`.

## Technical input

Feedback from @RobertIndie:

Regarding the configuration, there is no difference from [the original authorization provider.](https://pulsar.apache.org/docs/en/next/security-authorization/#enable-authorization-and-assign-superusers) You can just change the authorizationProvider . But currently it is only compatible with JWT authentication. It will try to get the role(or roles array) from the JWT token. For the case of roles array, it will check the permissions of all the roles and pass when one of them has the permission. Here is the PR for this feature: [#11341[Broker] Add multi roles support for authorization](https://github.com/apache/pulsar/pull/11341)

### PR DESCRIPTION
**Pull request by momo-jun**
### Motivation

Fix #14741 

### Modifications
Add doc for how to use MultiRolesTokenAuthorizationProvider.

Preview looks good.
image

### Documentation
  
- [x] `doc`

### CODE REVIEW

**Comment by momo-jun** [comment]
Hi @RobertIndie @codelipenghui @Anonymitaet can you pls review this PR? feel free to comment.

File: site2/docs/security-authorization.md [hunk 1]
```diff
@@ -98,3 +98,15 @@ PulsarAdmin admin = PulsarAdmin.builder()
                     .tlsTrustCertsFilePath("/path/to/trust/cert")
                     .build();
 ```
+
+## Authorize an authenticated client with multiple roles
+
+When a client is identified with multiple roles in a token (the type of role claim in the token is an array) during the authentication process, Pulsar supports to check the permissions of all the roles and further authorize the client as long as one of its roles has the required permissions.
+
+> Note: This authorization method is only compatible with [JWT authentication](security-jwt.md).
```
Comment (Anonymitaet) [inline site2/docs/security-authorization.md:106]:
```suggestion
> **Note**
> This authorization method is only compatible with [JWT authentication](security-jwt.md).
```

File: site2/docs/security-authorization.md [hunk 2]
```diff
@@ -98,3 +98,15 @@ PulsarAdmin admin = PulsarAdmin.builder()
...
+
+To enable the support, configure the authorization provider as `MultiRolesTokenAuthorizationProvider` in the `conf/broker.conf` file.
```
Comment (Anonymitaet) [inline site2/docs/security-authorization.md:108]: do we need to configure it in the `conf/standalone.conf` file? @RobertIndie

Comment (RobertIndie) [reply site2/docs/security-authorization.md:108]: We only need to do that when we use the standalone mode. But I think it's no need to mention it here.

**Review by RobertIndie — APPROVED** [review]

File: site2/docs/security-authorization.md (hunk 1 above)
Comment (momo-jun) [reply site2/docs/security-authorization.md]: Thanks @Anonymitaet. I've fixed this. PTAL.

**Comment by Anonymitaet** [comment]
@momo-jun any other doc versions should be applied the same changes?

**Comment by momo-jun** [comment]
> @momo-jun any other doc versions that should be applied to the same changes?

Yes, thanks for your reminder. Checked with @RobertIndie, I've added the doc for 2.8.1 and later versions.

**Comment by Anonymitaet** [comment]
> > @momo-jun any other doc versions that should be applied to the same changes?
> 
> Yes, thanks for your reminder. Checked with @RobertIndie, I've added the doc for 2.8.1 and later versions.

You can update 2.8.1 docs in this PR

**Comment by momo-jun** [comment]
> > > @momo-jun any other doc versions that should be applied to the same changes?
> > 
> > 
> > Yes, thanks for your reminder. Checked with @RobertIndie, I've added the doc for 2.8.1 and later versions.
> 
> You can update 2.8.1 docs in this PR

I've done that in my last commit. PTAL.

**Pull request merged**